common/config/babel/.babelrc.js

```diff
@@ -38,6 +38,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta' &&
         'one-to-n-calling',
         // PSTN calls 
         'PSTN-calls',
+        // raise hands feature
+        'raise-hands',
```

Comment:
small minor nit: 
Is the feature name "raise-hand" or "raise-hands" with an 's'? 
Looking below, I believe it's without the s.